src/controller/raid_floor.controller.ts

```diff
@@ -0,0 +1,115 @@
+import { UpdatedAt } from 'sequelize-typescript';
+import { Floor } from '../models/floor.model.js'
+
+
+///////Read or Pull all
+export const findAllFloor = async (req, res) => {
+    try {
+        console.log("Calling find all for Loot on the Table")
+        const allFloor:Floor[] = await Floor.findAll();
```

Comment:
Just look at comments for findAllLootTable, but this is surprisingly fine LOL

Reply:
Updated the messages messages sent with 200 and 500 status o7